website/docs/r/quicksight_ip_restriction.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+subcategory: "QuickSight"
+layout: "aws"
+page_title: "AWS: aws_quicksight_ip_restriction"
+description: |-
+  Manages the content and status of IP rules.
+---
+
+# Resource: aws_quicksight_ip_restriction
+
+Manages the content and status of IP rules.
+
+~> Deletion of this resource clears all IP restrictions from a QuickSight account.
+
+## Example Usage
+
+```terraform
+resource "aws_quicksight_ip_restriction" "example" {
+  enabled = true
+
+  ip_restriction_rule_map = {
+    "108.56.166.202/32" = "Allow self"
+  }
+
+  vpc_id_restriction_rule_map = {
+    (aws_vpc.example.id) = "Main VPC"
+  }
+}
+```
+
+## Argument Reference
+
+This resource supports the following arguments:
+
+* `enabled` - (Required) Whether IP rules are turned on.
+* `ip_restriction_rule_map` - (Optional) Map of allowed IPv4 CIDR ranges and descriptions.
+* `region` - (Optional) Region where this resource will be [managed](https://docs.aws.amazon.com/general/latest/gr/rande.html#regional-endpoints). Defaults to the Region set in the [provider configuration](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#aws-configuration-reference).
+* `vpc_endpoint_id_restriction_rule_map` - (Optional) Map of allowed VPC endpoint IDs and descriptions.
+* `vpc_id_restriction_rule_map` - (Optional) Map of VPC IDs and descriptions. Traffic from all VPC endpoints that are present in the specified VPC is allowed.
+
+## Attribute Reference
+
+This resource exports the following attributes in addition to the arguments above:
+
+* `aws_account_id` - The ID for the AWS account that contains the settings.
```

Comment:
Should this be in the argument reference?

Reply:
How about

> * `aws_account_id` - (Optional, Forces new resource) AWS account ID. Defaults to automatically determined account ID of the Terraform AWS provider.

as standard text?